### PR TITLE
[entt] Update to 3.11.0

### PR DESCRIPTION
**Pull request by Loghorn**
**Describe the pull request**

- #### What does your PR fix?

Updates EnTT to 3.11.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
No change to triplet support, tested on x64-windows

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

### CODE REVIEW

File: ports/entt/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "entt",
-  "version": "3.10.1",
+  "version": "3.11.0",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/entt/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/entt/vcpkg.json:4]: @Loghorn, Thanks for your pr, if you update the port version, please remove `port-version`

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/entt/vcpkg.json (hunk 1 above)
Comment (Loghorn) [reply ports/entt/vcpkg.json:4]: @JonLiu1993 if I remove `port-version`, should I also change the `baseline.json` and `entt.json` to remove it there too?

Comment (JonLiu1993) [reply ports/entt/vcpkg.json:4]: @Loghorn, You can remove port-version、baseline.json and entt.json then run command "./vcpkg x-add-version entt --overwrite-version" and commit again

File: versions/e-/entt.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
   "versions": [
+    {
+      "git-tree": "1bbb72320031821b14b973f3df4fe1555ac467dc",
+      "version": "3.11.0",
+      "port-version": 0
+    },
+    {
+      "git-tree": "3b10833e0c192da3fe85fbe23fc843fc0593a33a",
+      "version": "3.11.0",
+      "port-version": 1
```
Comment (JonLiu1993) [inline versions/e-/entt.json:11]:
```suggestion

```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by Loghorn** [comment]
> @Loghorn, when I tested the feature `experimental` , I got this error, Is this intentional?

@JonLiu1993 Yes: the experimental feature is intended to get the latest experimental branch from the EnTT repository, so it doesn't specify a commit or tag, but just the branch name; in this case vcpkg requires that you add the `--head` parameter to the command line: `vcpkg install entt[experimental] --head`

**Comment by JonLiu1993** [comment]
Feature `experimental` is tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by JavierMatosD** [comment]
Thank you!

**Pull request merged**